srcpkgs/hdf5/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'hdf5'
 pkgname=hdf5
-version=1.10.5
+version=1.12.0
 revision=1
+_version=1.12
 build_style=gnu-configure
 configure_args="--enable-cxx --enable-fortran"
 hostmakedepends="gcc-fortran"
 makedepends="zlib-devel"
 short_desc="Data model, library, and file format for storing and managing data"
 maintainer="pulux <pulux@pf4sh.de>"
-license="custom"
+license="custom:hdf5"
 homepage="https://hdfgroup.org"
-distfiles="https://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-${version}.tar.gz"
-checksum=6d4ce8bf902a97b050f6f491f4268634e252a63dadd6656a1a9be5b7b7726fa8
+distfiles="https://support.hdfgroup.org/ftp/HDF5/releases/${pkgname}-${_version}/${pkgname}-${version}/src/${pkgname}-${version}.tar.gz"
```

Comment:
could you use `${version%.*}` instead of _version?

Reply:
this could be a good time to add a changelog:
```
changelog="https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-${version%.*}/hdf5-${version}/src/hdf5-${version}-RELEASE.txt"
```